### ISSUE TITLE
Hold key instead of tapping

### ISSUE THREAD

**Issue by Underscore85** [superduper/BrightnessMenulet#93]
Hi,

It would be nice if you hold the brightness up/down key, the value keeps changing untill you release the key (just like Mac screen). Now I have to tap the button. 

I'm using a LG 27UD88-W with a Macbook Pro 2016 and a Magic Keyboard. I use Karabiner Elements to map the brightness keys (f1/f2) to f16/f17. Works like a charm 👍 

Hope you can create this! 

Thank you!

Cheers,
Underscore85

**Comment by d14b0ll0s** [comment]
I have exactly the same setup. You can also use an app Alfred and its 'Workflows' to map the keys to specific commands, such as ddcctl (command line), which I'm using to change brightness to predifined values (such as /usr/local/bin/ddcctl -d 1 -b 10).

Have you found a way to use just F1 / F2 for +/- 5% brightness? ddcctl wouldn't recognise LG's instructions properly, as opposed to Dell's, so I'm only able to set it to a specific value.

**Comment by Underscore85** [comment]
Thanks, but I think the solution you propose sets a specific brightness level when you press the key. I'm looking for simular functionality as a normal Mac display, like holding the key keeps changing the level.

I remapped the F1/F2 buttons with Karabiner Elements as F16/F17. In the BrightnessMenulet I mapped them as brightness controls. In that way it works. No FN key nessesary :) Only not the fluent way...

**Comment by d14b0ll0s** [comment]
So does it work the same way as with normal Mac brightness controls and it remembers the last value? I need to give it a try again. Is it via Thunderbolt 3 / USB-C? (unfortunately I damaged mine and pulled it out so I'm on TB2 now)

**Comment by d14b0ll0s** [comment]
For me, Brightness Menulet does properly not work on LG 27UD88-W. I can't move the sliders and it doesn't recognise the current brightness value.

Edit: screenshot attached.
![screen shot 2018-02-28 at 11 34 05](https://user-images.githubusercontent.com/5240123/36783107-58369096-1c7b-11e8-8138-4613174306df.jpg)

**Comment by d14b0ll0s** [comment]
![screen shot 2018-02-28 at 11 33 15](https://user-images.githubusercontent.com/5240123/36783064-38e64d76-1c7b-11e8-9555-4599652b7cdf.jpg)

**Comment by Underscore85** [comment]
Yes it works properly. I connected via USB-C, but I use a Belkin Thunderbolt 3 HD hub in between, because the display and Macbook together are a nightmare with waking up and stuff. 

Here's what I did: https://www.reddit.com/r/mac/comments/6kj0oh/use_the_real_brightness_keys_on_a_real_apple/

Have you also got the newest firmware on your display?
